Themis/Views/Authentication/AuthenticationView.swift

```diff
@@ -20,12 +20,13 @@ struct AuthenticationView: View {
                 .padding()
             TextField("Artemis-Server", text: $authenticationVM.serverURL)
                 .textFieldStyle(LoginTextFieldStyle())
```

Comment:
would also disable autocapitalization here, if we're already at it

Reply:
Good idea! Apparently, you can also tell iOS that a text field is an URL, so I did that.